examples/realworld/ext/article/components/ArticleEditorPage.js

```diff
@@ -44,7 +43,7 @@ const ArticleEditorPage = (props) => {
   //   When is article null, when not, should I look into combination of article and articleSlug, then
   //   there is this 'enabled' which I need on the other hand -> uff. And what if I get error? humpf!
   const articleSlug = props.match.params.articleSlug
-  const { data: article, error: articleError } = useQuery(getArticle, { slug: articleSlug }, { enabled: articleSlug })
+  const { data: article, error: articleError } = useQuery(getArticle, { slug: articleSlug }, { enabled: !!articleSlug })
```

Comment:
This fixes [the 500 error @shayneczyzewski noticed](https://github.com/wasp-lang/wasp/pull/554#issuecomment-1095268143).

Reply:
Uff nice catch!

Reply:
Awesome find and fix, thanks! 👏🏻